### PR TITLE
OPT: Use environ package instead of dotenv

### PR DESCRIPTION
**Pull request by saeedghx68**
I've searched on the internet, I think [this package](https://github.com/joke2k/django-environ) is much better than `dotenv` package

### CODE REVIEW

File: src/proj/settings/base.py [hunk 1]
```diff
@@ -13,14 +13,22 @@
 import os
 from pathlib import Path
 
+import environ
+
```
Comment (hramezani) [inline src/proj/settings/base.py:17]: Please remove this line

**Comment by hramezani** [comment]
I think we need to fetch env variables with `ENV_SETTING` function in https://github.com/mirzakhany/django_boilerplate/blob/master/src/proj/settings/base.py#L26

Also, we can load other env variables like `DEBUG`, `ALLOWED_HOSTS`, and ... by `environ`

**Review by saeedghx68 — COMMENTED** [review]
> I think we need to fetch env variables with `ENV_SETTING` function in https://github.com/mirzakhany/django_boilerplate/blob/master/src/proj/settings/base.py#L26
> 
> Also, we can load other env variables like `DEBUG`, `ALLOWED_HOSTS`, and ... by `environ`

Done Hasan

File: src/proj/settings/base.py (hunk 1 above)
Comment (hramezani) [reply src/proj/settings/base.py:17]: Please remove the blank line after `import environ`

**Review by hramezani — APPROVED** [review]
:+1:

**Review by mirzakhany — APPROVED** [review]

**Review by peymanslh — APPROVED** [review]

**Pull request merged**